README.md

```diff
@@ -13,11 +13,11 @@ check out our [PythonAnywhere demo site](http://defectdojo.pythonanywhere.com/).
 
 You can log in as an administrator like so:
 
-![Admin](https://inventropy.us/dd/admin.png) with password **administrator**
+![Admin](https://raw.githubusercontent.com/OWASP/django-DefectDojo/master/doc/img/admin.png)
```

Comment:
We should just use markdown here, instead of images. I also recommend a line break between username and password. Use backticks or something similar to set the apart from the regular doc flow. Thanks!
`username: admin`
`password: administrator`